modules/ocf_mirrors/manifests/kde.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_mirrors::kde {
+  file {
+    '/opt/mirrors/project/kde':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/kde/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  cron {
+    'kde':
+      command => '/opt/mirrors/project/kde/sync-archive > /dev/null',
+      user    => 'mirrors',
+      hour    => '*/2',
```

Comment:
I think putting `minute` as `0` explicitly here is a good idea, just to make it clear.